docs/main/updating/5-0.md

```diff
@@ -71,8 +87,8 @@ cordovaAndroidVersion = '10.1.1'
 
     dependencies {
         classpath 'com.android.tools.build:gradle:7.4.1'
```

Comment:
this is actually updated from 7.2.1 to 7.4.1 on next step, so we should probably change the order, because at this point users would have 7.2.1, that or replace 7.4.1 to 7.2.1 since it will be then updated on next step